src/command/commands.rs

```diff
@@ -182,11 +182,17 @@ impl Command for CreatePlaceholders<'_> {
     fn build(&self) -> CF_OPERATION_PARAMETERS_0 {
         CF_OPERATION_PARAMETERS_0 {
             TransferPlaceholders: CF_OPERATION_PARAMETERS_0_7 {
-                Flags: CloudFilters::CF_OPERATION_TRANSFER_PLACEHOLDERS_FLAG_NONE,
+                Flags: CloudFilters::CF_OPERATION_TRANSFER_PLACEHOLDERS_FLAG_DISABLE_ON_DEMAND_POPULATION,
                 CompletionStatus: Foundation::STATUS_SUCCESS,
                 PlaceholderTotalCount: self.total as i64,
-                PlaceholderArray: self.placeholders.as_ptr() as *mut _,
-                PlaceholderCount: self.placeholders.len() as u32,
+                PlaceholderArray: self
+                    .placeholders
+                    .map(|placeholders| placeholders.as_ptr() as *mut _)  // FIXME
+                    .unwrap_or_else(|| ptr::null_mut()),
+                PlaceholderCount: self
+                    .placeholders
+                    .map(|placeholders| placeholders.len() as u32)
+                    .unwrap_or(0),
                 EntriesProcessed: 0,
```

Comment:
Are we sure this doesn't need to be set? What's the point of it? The docs here: https://learn.microsoft.com/en-us/windows/win32/api/cfapi/ne-cfapi-cf_operation_transfer_placeholders_flags don't even mention use of that parameter.

Reply:
I think it's actually an output value, similar to here: https://learn.microsoft.com/en-us/windows/win32/api/cfapi/nf-cfapi-cfcreateplaceholders

Reply:
This doesn't make sense, EntriesProcessed in CF_OPERATION_PARAMETERS is DWORD instead of PDWORD.

Reply:
`TransferPlaceholders.PlaceholderTotalCount <= (Sum of Prior TransferPlaceholders.EntriesProcessed) + Current TransferPlaceholders.PlaceholderCount`

So, I guess by "sum of prior ..." they mean the currently specified `EntriesProcessed`? Which wouldn't matter if we are specifying `CF_OPERATION_TRANSFER_PLACEHOLDERS_FLAG_DISABLE_ON_DEMAND_POPULATION` anyways, like you said